### ISSUE TITLE
Courses: View progress from the dropdown menu.  horizontal Scroll bar

### ISSUE THREAD

**Issue by chtelmko** [open-learning-exchange/planet#6728]
Should there be the horizontal scroll bar? 
![course progress detail](https://user-images.githubusercontent.com/40531136/86366990-7af91480-bc49-11ea-8d7e-0af6f488c059.png)

**Comment by paulbert** [comment]
Yes and no.  There should be scrolling in this view, but the scrollbars should be hidden.  Unfortunately the scrollbars aren't hidden in Firefox.

I'm going to leave this open as **difficult** since I'm not sure there's a way to maintain functionality without the scrollbars in this view.
